test-coverage.txt

```diff
@@ -0,0 +1,54 @@
+> react-scripts test --env=jest-environment-jsdom-sixteen "--coverage" "--watchAll=false"
+
+ PASS  src/tests/Fortune.test.jsx
+ PASS  src/tests/Header.test.jsx
+ PASS  src/tests/App.test.jsx
+ PASS  src/tests/MainMenu.test.jsx
+ PASS  src/tests/VideoHome.test.jsx
+ PASS  src/tests/VideoItem.test.jsx
+ PASS  src/tests/UserMenu.test.jsx
+ PASS  src/tests/Search.test.jsx
+ PASS  src/tests/globals.test.js
+ PASS  src/tests/DarkMode.test.jsx
+--------------------------|----------|----------|----------|----------|-------------------|
+File                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
+--------------------------|----------|----------|----------|----------|-------------------|
+All files                 |      100 |      100 |      100 |      100 |                   |
```

Comment:
Although the test coverage is 100%, I'd like to see at least one test for the events using the fire event, and the render function to query the screen instead of the snapshot